src/browser/css/components/TabBar.css

```diff
@@ -37,7 +37,3 @@
   margin-top: 5px;
   border-radius: 50%;
 }
-
-.TabBar .teamTabItem-label {
-  font-weight: bold;
```

Comment:
`font-weight: bold` must be used only in the case where the guest page has unread channels. In the meaning of that, the original `.teamTabItem-label` is not appropriate class name though. So feel free to edit class name if you need.

Reply:
Pushed changes to address that.